src/dotnet/commands/dotnet-pack/PackCommand.cs

```diff
@@ -17,37 +17,37 @@ public static int Run(string[] args)
             CommandLineApplication cmd = new CommandLineApplication(throwOnUnexpectedArg: false)
             {
                 Name = "pack",
-                FullName = "pack",
-                Description = "pack for msbuild",
+                FullName = LocalizableStrings.AppFullName,
+                Description = LocalizableStrings.AppDescription,
                 AllowArgumentSeparator = true,
                 ArgumentSeparatorHelpText = HelpMessageStrings.MSBuildAdditionalArgsHelpText
             };
 
             cmd.HelpOption("-h|--help");
 
-            var output = cmd.Option("-o|--output <OUTPUT_DIR>",
-                "Directory in which to place outputs",
+            var output = cmd.Option($"-o|--output <{LocalizableStrings.CmdOutputDir}>",
```

Comment:
NIT: I would prefer that when you put parameters in a new line that each of them have their own new line. Line this:

cmd.Option(
    $"...",
    Localiza...,
    CommandOptionType...);